pkgs/servers/web-apps/shiori/default.nix

```
@@ -2,17 +2,17 @@
 
 buildGoModule rec {
   pname = "shiori";
-  version = "1.5.5";
+  version = "1.6.2";
 
-  vendorHash = "sha256-suWdtqf5IZntEVD+NHGD6RsL1tjcGH9vh5skISW+aCc=";
+  vendorHash = "sha256-LLiBRsh9HsadeHQh4Yvops1r2GfjtvQKt5ZelQnPGdI=";
 
   doCheck = false;
 
   src = fetchFromGitHub {
     owner = "go-shiori";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-kGPvCYvLLixEH9qih/F3StUyGPqlKukTWLSw41+Mq8E=";
+    sha256 = "sha256-1ZZpQXlAHs5MTowCv3sWS3L7X5FTnU/b4trvHPiz+uE=";
   };
 
   passthru.tests = {
```

```suggestion
  nativeBuildInputs = [ installShellFiles ];

  postInstall = ''
    installShellCompletion --cmd shiori \
      --bash <($out/bin/shiori completion bash) \
      --fish <($out/bin/shiori completion fish) \
      --zsh <($out/bin/shiori completion zsh)
  '';

  passthru.tests = {
```
Make sure to add `installShellFiles` to the top.

---

Thank you so much! I didn't know about this, and I implemented it as you suggested, and it works!